apps/src/netsim/NetSimLogBrowserTable.jsx

```diff
@@ -0,0 +1,170 @@
+/** @file Table of log rows displayed in the log browser */
+import React from 'react';
+import orderBy from 'lodash/orderBy';
+import {Table, sort} from 'reactabular';
+import moment from 'moment';
+import Packet from './Packet';
+
+const style = {
+  nowrap: {
+    whiteSpace: 'nowrap'
+  },
+  prewrap: {
+    whiteSpace: 'pre-wrap'
+  }
+};
+
+/**
+ * Table of log rows displayed in the Log Browser modal.
+ * Wraps configuration and sorting behavior around a Reactabular table.
+ * @see http://reactabular.js.org
+ */
+const NetSimLogBrowserTable = React.createClass({
+  propTypes: {
+    logRows: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
+    headerFields: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
+    renderedRowLimit: React.PropTypes.number
+  },
+
+  getInitialState() {
+    return {
+      sortingColumns: {
+        0: {direction: 'desc', position: 0}
+      }
+    };
+  },
+
+  render() {
+    const headerFields = this.props.headerFields;
+
+    // Define a sorting transform that can be applied to each column
+    const sortable = sort.sort({
```

Comment:
[Copypasta](http://reactabular.js.org/#/sorting).

Actually, it'd be pretty clean to put both `getSortingColumns` and `onSort` onto the component, since they'll be implicitly bound.  Great suggestion!